blocks/overline-block/intl.json

```diff
@@ -1,13 +1,13 @@
 {
 	"overline-sponsored-content": {
-		"de": "Werblicher Inhalt",
+		"de": "Sponsored Content",
```

Comment:
interesting this looks like a regression

Reply:
We can take the existing values and add to lokalise then regenerate the locale files to avoid this regression

Reply:
oh ok

Reply:
it seems like lokalise may have an issue ... trying to see what's happened there

Reply:
cool can manually update lokalise with this

Reply:
updated to overline